service/openapi.yaml

```diff
@@ -329,6 +329,20 @@ components:
           properties:
             cacheInvalidation:
               $ref: '#/components/schemas/featureCacheInvalidation'
+            evaluation:
+              $ref: '#/components/schemas/featureEvaluation'
+    featureEvaluation:
```

Comment:
suggesting to rename to flagEvaluation, mapping the OF spec [1]

[1] - https://openfeature.dev/specification/sections/flag-evaluation#13-flag-evaluation

Reply:
Please also update the ref to use the updated name.